tests/solvers/quadratic/fgw_test.py

```diff
@@ -250,7 +250,7 @@ def test_fgw_lr_memory(self, rng: jnp.ndarray, jit: bool):
   def test_fgw_lr_generic_cost_matrix(
       self, rng: jnp.ndarray, cost_rank: Union[int, Tuple[int, int, int]]
   ):
-    n, m = 70, 100
+    n, m = 20, 30
```

Comment:
you should maybe raise an issue here to say that it bugs with bigger dims.

Reply:
bugs -> doesn't converge